jupiter-tests/src/test/java/org/junit/jupiter/params/converter/DefaultArgumentConverterTests.java

```diff
@@ -103,43 +94,14 @@ void throwsExceptionForNonStringsConversion() {
 		verify(underTest, never()).convert(any(), any(), any(ClassLoader.class));
 	}
 
-	@Test
-	void delegatesStringsConversion() {
-		doReturn(null).when(underTest).convert(any(), any(), any(ClassLoader.class));
-
-		convert("value", int.class);
-
-		verify(underTest).convert("value", int.class, getClassLoader(DefaultArgumentConverterTests.class));
-	}
-
-	@Test
-	void convertsLocaleWithDefaultFormat() {
-		when(context.getConfigurationParameter(eq(DEFAULT_LOCALE_CONVERSION_FORMAT_PROPERTY_NAME), any())) //
-				.thenReturn(Optional.empty());
-
-		assertConverts("en", Locale.class, Locale.ENGLISH);
-		assertConverts("en-US", Locale.class, Locale.US);
-	}
-
-	@Test
-	void convertsLocaleWithExplicitBcp47Format() {
-		when(context.getConfigurationParameter(eq(DEFAULT_LOCALE_CONVERSION_FORMAT_PROPERTY_NAME), any())) //
-				.thenReturn(Optional.of(BCP_47));
-
-		assertConverts("en", Locale.class, Locale.ENGLISH);
-		assertConverts("en-US", Locale.class, Locale.US);
-	}
-
-	@Test
-	void delegatesLocaleConversionWithExplicitIso639Format() {
-		when(context.getConfigurationParameter(eq(DEFAULT_LOCALE_CONVERSION_FORMAT_PROPERTY_NAME), any())) //
-				.thenReturn(Optional.of(ISO_639));
-
+	@ParameterizedTest
+	@ValueSource(classes = { int.class, Locale.class })
```

Comment:
Nitpicking, now that `DefaultArgumentConverter` doesn't have a type-specific logic, is keeping a `Locale` test case valuable?

Reply:
You have a point. Removed in a75932e22d3c12dd2bc4489f822d88437f509b09.